index.html

```diff
@@ -94,7 +102,7 @@ <h2 class="heading-underline">Upcoming events</h2>
 					<div class="articles row">
 						<article class="article">
```

Comment:
You cannot have a `row` without `col-...` classes on the child elements.

Reply:
I'll do it again.
Thank you for your review.